webapp/configs/limenet.py

```diff
@@ -1,11 +1,23 @@
 WEBAPP_CONFIG = {
     'LAYOUT': '_layout-brandstore.html',
-    'STORE_NAME': 'Limestore',
+    'STORE_NAME': 'LimeNET',
     'STORE_BRAND': {
-        'LOGO': None,
-        'COLOUR': None
+        'LOGO': 'https://assets.ubuntu.com/v1/cc2b2033-lime-logo.svg',
+        'COLOUR': '#fff'
     },
-    'STORE_INSTALL_INSTRUCTIONS': False,
-    'FOOTER_TEXT': None,
+    'STORE_LINKS': [
+        {
+            'title': 'Lime Micro',
+            'href': 'http://www.limemicro.com/'
+        },
+        {
+            'title': 'Myriad-RF',
+            'href': 'http://www.limemicro.com/'
+        }
+    ],
+    'CUSTOM_STYLES': '.p-navigation__link a {color: #000;}\
```

Comment:
As much as it probably works in short term, CUSTOM_STYLES should be a URL or file name to CSS file.

Reply:
Agreed, ideally we wouldn't do any 'custom styles' at all (just config for colours) but at this point i don't want to make things too complicated as Lime might say they hate it and don't want any custom styles :D!